### PR TITLE
[IOTDB-3840]  deserialize RemoveDataNodePlan exception

### PR DESCRIPTION
**Pull request by BanDuWuZi**
## Description
when remove datanode, the follower confignodes has exception in error log file, remove does not succeed

##### Key changed/added classes (or packages if there are too many classes) in this PR
RemoveDataNodePlan，RemoveDataNodePlanTest

### CODE REVIEW

File: confignode/src/test/java/org/apache/iotdb/confignode/consensus/request/write/RemoveDataNodePlanTest.java [hunk 1]
```diff
@@ -129,23 +129,44 @@ public void testTestHashCode() {
...
+    }
+  }
+
+  @Test
+  public void testUpdatePlanSerializeAndDeSerialize() {
+    try {
+      RemoveDataNodePlan deSerializeReq = runPlanSerializeAndDeSerialize(true);
+      Assert.assertEquals(req1, deSerializeReq);
+    } catch (IOException e) {
+      // do nothing
```
Comment (wangchao316) [inline confignode/src/test/java/org/apache/iotdb/confignode/consensus/request/write/RemoveDataNodePlanTest.java:147]: add fail(e.getMessage());

File: confignode/src/test/java/org/apache/iotdb/confignode/consensus/request/write/RemoveDataNodePlanTest.java [hunk 2]
```diff
@@ -129,23 +129,44 @@ public void testTestHashCode() {
   }
 
   @Test
-  public void testSerializeAndDeSerialized() throws IOException {
+  public void testNotUpdatePlanSerializeAndDeSerialize() {
+    try {
+      RemoveDataNodePlan deSerializeReq = runPlanSerializeAndDeSerialize(false);
+      Assert.assertEquals(req1, deSerializeReq);
+    } catch (IOException e) {
+      // do nothing
```
Comment (wangchao316) [inline confignode/src/test/java/org/apache/iotdb/confignode/consensus/request/write/RemoveDataNodePlanTest.java:137]: add fail(e.getMessage());

File: confignode/src/test/java/org/apache/iotdb/confignode/consensus/request/write/RemoveDataNodePlanTest.java (hunk 1 above)
Comment (BanDuWuZi) [reply confignode/src/test/java/org/apache/iotdb/confignode/consensus/request/write/RemoveDataNodePlanTest.java]: fixed

File: confignode/src/test/java/org/apache/iotdb/confignode/consensus/request/write/RemoveDataNodePlanTest.java (hunk 2 above)
Comment (BanDuWuZi) [reply confignode/src/test/java/org/apache/iotdb/confignode/consensus/request/write/RemoveDataNodePlanTest.java]: fixed

**Review by wangchao316 — APPROVED** [review]
LGTM

**Pull request merged**